src/pages/Main.js

```diff
@@ -15,11 +16,14 @@ import IdleStatusMonitor from 'src/components/IdleStatusMonitor'
 import ImportStatus from 'src/components/ImportStatus'
 import SupportRequest from 'src/components/SupportRequest'
 import { getOidcConfig } from 'src/libs/auth'
+import { isAxeEnabled } from 'src/libs/config'
 import { PageViewReporter } from 'src/libs/events'
 import { LocationProvider, PathHashInserter, Router, TitleManager } from 'src/libs/nav'
 import { AuthenticatedCookieSetter } from 'src/pages/workspaces/workspace/analysis/runtime-common'
 
 
+const AxeCore = lazy(() => import('src/components/AxeCore'))
```

Comment:
Is it necessary to use React for this? Would this work without the AxeCore component, lazy, and Suspense?

```js
if (isAxeEnabled()) {
  import('src/components/AxeCore')
}
```

Reply:
I recalled Isaac telling me that just gating the import wasn't sufficient to prevent the library from getting bundled up into our 3rd party libraries, so I decided to convert it to a component and use the lazy loading. But locally at the source files in production, I don't see that axe-core is getting included.

Therefore I will revert back to the simpler way that I had this configured before. I'll put the PR in draft while I refactor.

Reply:
The dynamic import should split it off into a separate chunk.
https://webpack.js.org/guides/code-splitting/#dynamic-imports